application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableRoleBasedAction.java

```diff
@@ -0,0 +1,50 @@
+package org.togetherjava.tjbot.commands.moderation.temp;
+
+import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link RevocableModerationAction} abstract implementation for a role based moderation action.
+ * <br />
+ * <br />
+ * Implements {@link RevocableModerationAction#handleRevokeFailure(ErrorResponse, long)} to handle
+ * possible errors for a role-based revocation action.
```

Comment:
A detail that shouldnt be part of the javadoc since it might only be true "today" but not tomorrow.

Reply:
removed it.